config/defaults_template.yaml

```diff
@@ -1,8 +1,9 @@
 # See ./defaults.go for documentation on each field.
 
-driverPool: "{{ .DriverPool }}"
-
-workerPool: "{{ .WorkerPool }}"
+defaultPoolLabels:
+  client: default-client-pool
+  driver: default-driver-pool
+  server: default-server-pool
```

Comment:
I think it may be better to make these like LabelSelectors. Instead of relying on a hardcoded "true" string... maybe we should make this specify both a key and a value. wdyt?

Reply:
Reading the documentation, that makes sense:

https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/

So, if the pool is not specified, a label selector is added instead.

One concern I would have is that the implementation might be more complex, but perhaps not.

If it takes time to do, it can be done in a follow-up PR, since it does not change the interface.

Reply:
I misread the documentation in the comment above. The label always has a value, so the most we should do is make the value a constant in our code, instead of hardcoding to "true" (since it appears in three places).

We should not change the interface to require the user to specify a value, because that would make life more difficult for the user.